gammapy/data/observation_stats.py

```diff
@@ -0,0 +1,217 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import astropy.units as u
+from ..stats import Stats
+from gammapy.stats import significance_on_off
+
+__all__ = [
+    'ObservationStats',
+    'ObservationStatsList'
+]
+
+
+class ObservationStats(Stats):
+    """Observation statistics.
+
+    Class allowing to summarize observation 
+    (`~gammapy.data.DataStoreObservation) statistics
```

Comment:
One more fail on travis-ci in the docs build:
https://travis-ci.org/gammapy/gammapy/jobs/137211316#L1660

```
/home/travis/build/gammapy/gammapy/build/lib.linux-x86_64-2.7/gammapy/data/observation_stats.py:docstring of gammapy.data.ObservationStats:5: WARNING: Inline interpreted text or phrase reference start-string without end-string.
```
There's a missing backtick here, it should be:
```
(`~gammapy.data.DataStoreObservation`) statistics
```